test/transfer-manager.ts

```diff
@@ -255,10 +299,18 @@ describe('Transfer Manager', () => {
       }
     }
 
-    beforeEach(() => {
-      readStreamStub = sandbox
-        .stub(fs, 'createReadStream')
-        .returns(pThrough as unknown as fs.ReadStream);
+    before(async () => {
+      directory = await fsp.mkdtemp(
+        path.join(tmpdir(), 'tm-uploadFileInChunks-')
+      );
+
+      filePath = path.join(directory, 't.txt');
+
+      await fsp.writeFile(filePath, 'hello');
```

Comment:
Sharing a real, temp file to make it more robust. We can refactor the other tests if we want to use the real XML tool w/o exporting it.

Reply:
Yea this looks better than the hacky way I had it.